### ISSUE TITLE
Chrome Profile Support

### ISSUE THREAD

**Issue by jokull** [AlexStrNik/Browserino#4]
Have you considered Chrome profile support? That would be cool. BrowserOpener supports this. Excited to try this as an alternative.

**Comment by AlexStrNik** [comment]
@dgimb89 Yeah, I actually already figured out how to implement this technically. The only reason I haven’t done it yet is that I’m still working on a major redesign of the Rules, Apps, and Browsers settings.

Originally, this project was just a Browserosaurus clone with a simple fix for the spaces issue, so I hadn’t planned for these additional features. But after multiple requests, the functionality has expanded—along with some overlaps, duplications, and UI complexity. For example, you can configure non-browser apps to appear in the list for certain domains in one tab, but to open them instantly, you need to add a regex in another tab. Adding yet another tab just to configure Chrome profiles doesn’t feel like the right solution.

One day, I’ll find the inspiration to design a better way to manage browsers, rules, and apps—and I’ll implement it. For now, if you really need this feature, someone has already added it in their fork: https://github.com/mstfe/Browserino.

**Comment by AlexStrNik** [comment]
@xa4hf8 check where is Chrome or Brave installed, maybe you need to add a new path in Locations tab

**Comment by eBeyond** [comment]
This issue would be great to be solved. But not only for chrome. I think it would be helpful for other browsers (chromium based) as well. (Like vivaldi)
For me it would be totally ok, if it would be possible to have the possibility to have two chromes/vivalid where I can pass the argument to start profile 1 or 2. That would be great.

**Comment by dan-proudfoot-bots** [comment]
I'm trying to slim down my browser usage and it would be really cool to get profile support. The new version looks awesome and it would be really cool to see it in action!

**Comment by louishamelers** [comment]
Any update on when this will be released @AlexStrNik ?

**Comment by jasonliaotw** [comment]
Looking forward to this feature being implemented soon, thanks.
